static-assets/scripts/resources/locale-de.json

```diff
@@ -330,28 +330,26 @@
       "LEVEL_ERROR": "error"
     },
     "logconsole": {
-      "HEADER_LABEL": "Protokollkonsole",
+      "HEADER_LABEL": "Logging Konsole",
       "TIMESTAMP": "Zeitstempel",
-      "MESSAGE": "Botschaft",
-      "DETAILS": "Einzelheiten",
-      "CLEAR": "Klar",
-      "LEVEL": "Niveau",
-      "THREAD": "Faden",
-      "SITE": "Seite",
-      "EXCEPTION": "Ausnahme"
+      "MESSAGE": "Message",
+      "DETAILS": "Details",
+      "CLEAR": "Zurücksetzen",
+      "LEVEL": "Level",
+      "THREAD": "Thread",
+      "SITE": "Projekt",
+      "EXCEPTION": "Exception"
     }
   },
   "common": {
     "CANCEL_LABEL": "Abbrechen",
     "SEARCH_LABEL": "Suchen",
     "CLOSE_LABEL": "Schließen",
     "ADD_LABEL": "Hinzufügen",
-    "CREATE": "Erstellen",
+    "CREATE": "create",
```

Comment:
Please confirm this is the correct capitalization. Same for the “edit” below.

Reply:
These are the "technical names" of the dialogs. They need to be consistent through all languages, e.g. `<div class="form-group" ng-if="!(dialogMode == 'create')"> `

(I think they should not be part of the resource bundles)

Reply:
Could you point me to where in the code have you seen a message bundle item used to be compared as a constant? Indeed that shouldn’t happen. 

However, I’d think these items (Create and Edit) are rather used to title a dialog (e.g. in the header of a pop up) or something of sorts. Actual text on the UI, in which case they should also keep the capitalisation and should be translated.

Reply:
https://github.com/craftercms/studio-ui/blob/master/static-assets/ng-views/admin-users.html#L95
`dialogMode` in here is taken from the resource bundle. So in this case, any other value than `create` would break the user creation dialog, see:

<img width="1838" alt="dialogMode" src="https://user-images.githubusercontent.com/1357798/63542144-f402e800-c51f-11e9-96ab-41ed794d9be8.png">

Reply:
This is a bug! Thanks for pointing it out.
https://github.com/craftercms/craftercms/issues/3388